### ISSUE TITLE
When export and import a collection the request inside the folders are put outsite

### ISSUE THREAD

**Issue by richardblynd** [a85/POSTMan-Chrome-Extension#2143]
1. Postman Version: **4.3.3**
2. App (Chrome app or Mac app): **Chrome 54.0.2704.103**
3. OS details: **Windows 7 64 bits**
4. Is the Interceptor on and enabled in the app: **No**
5. Did you encounter this recently, or has this bug always been there: **Noticed it today 6/21/2016**
6. Expected behaviour: **When import a collection all data is put out of folders.**
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app): **no**
8. Screenshots (if applicable) **no**

Steps to reproduce the problem:
1. Create a collection and a folder
2. Create a request inside the folder
3. Export the collecion
4. Import the collection saved before

I think the problem is in below method, if the item is not a folder then the method return undefined

```
order: function (collectionV2) {
    var itemArray = collectionV2.items || collectionV2.item,
        allItems = _.isArray(itemArray) ? itemArray : [itemArray];
    return _.filter(_.map(allItems, function (item) {
        var isFolder = (item.items || item.item);
        if (!isFolder) {
            return item.id || item._postman_id;
        }   
    }));
}
```

**Comment by Aaronphall** [comment]
I have ran into the same issue and accidently discovered a workaround.

If someone is using version 4.3.2 the folder structure will stay correct. Have validated that a user can import from a version 4.3.3 created collection and as long as they have version 4.3.2, they will have the correct structure.

**Comment by abhijitkane** [comment]
@Aaronphall @richardblynd Thanks for reporting this. You can use the v1 export format for now. We'll be pushing a fix for this in the next release.

**Comment by arushi-pant** [comment]
I am facing the same issue with both v1 & v2 export formats in Postman v4.3.3. 
How do I downgrade postman version to 4.3.2?

**Comment by abhijitkane** [comment]
@arushi-pant Not sure how you're seeing this with v1. That part of the app hasn't changed in years, and I'm not able to reproduce this. Can you duplicate the collection and try exporting as v1?

There's currently no way to downgrade app versions. We'll be pushing the fix for the v2 issue in the next release.

**Comment by abhijitkane** [comment]
@arushi-pant @Aaronphall @richardblynd This has been fixed with v4.4.1 of the app. You can force an upgrade (https://www.getpostman.com/docs/updating)
